src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/integration/MultiNodeTestBase.kt

```diff
@@ -21,21 +21,23 @@ import java.time.Duration
 
 abstract class MultiNodeTestBase {
 
+  internal val hmppsQueueService = getBean("hmppsQueueService", HmppsQueueService::class.java)
+
+  internal fun <T> getBean(name: String, clazz: Class<T>) =
```

Comment:
Should this be 'class' instead of 'clazz'?

Reply:
it's a convention, `class` is part of the syntax and can't be a variable name